docs/specs/config.yml

```
@@ -493,3 +493,38 @@ outputs:
   .publish.json: |
   .errors.log: |
       ["error","publish-url-conflict","Two or more files publish to the same url '/a.png': 'a.png', 'docs/a.png'"]
+---
+# Redirection files have conflicted ids
+# all redirection files will be built
+# redirection document id is retrieved from the first redirection file (a.md for below case)
+inputs:
+  docfx.yml: |
+    redirections:
+      a.md: /docs/path1
+      b.md: /docs/path1
+  docs/path1.md:
+outputs:
+  .errors.log: |
+    ["warning","redirection-url-conflict","The '/docs/path1' appears twice or more in the redirection mappings","docfx.yml",3,9]
+  .publish.json: |
+    {
+        "files": [
```

consider condense output expectation to only include what must be verified:

```suggestion
        "files": [
          { "url": "/a", "redirect_url": "/docs/path1" },
          { "url": "/b", "redirect_url": "/docs/path1" },
          { "url": "/docs/path1" },
```

---

considered, the reason why we fix it is we missed one of them in the publish items